clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -1648,6 +1648,26 @@ private static void checkMigrationStatus(CommandLine cmd) {
     ControllerClient destControllerClient = new ControllerClient(destClusterName, veniceUrl, sslFactory);
 
     ChildAwareResponse response = srcControllerClient.listChildControllers(srcClusterName);
+
+    // Check the migration status for system stores in the source and destination clusters
+    MultiStoreResponse srcClusterStores = srcControllerClient.queryStoreList();
+    for (String currStoreName: srcClusterStores.getStores()) {
+      VeniceSystemStoreType systemStoreType = VeniceSystemStoreType.getSystemStoreType(currStoreName);
+
+      if (systemStoreType != null && currStoreName.matches(systemStoreType.getSystemStoreName(currStoreName))) {
```

Comment:
Other tweak.  This if statement never evaluates to true.  I think it's because currStoreName is a larger string, and systemStoreType.getSystemStoreName(currStoreName) does... I'm not sure, but probably something which is a trim of currStoreName.

I think we can tweak this to be: systemStoreType.extractRegularStoreName(currStoreName).equals(storeName)

This I think would also avoid gotchas where theres a storeName which is a substring of a different store.